array-manipulation-2/src/drop-right.js

```diff
@@ -2,9 +2,7 @@
 function dropRight(array, count) {
   var newArray = [];
   if (count > array.length) {
-    for (var i = 0; i < array.length; i++) {
-      newArray.push(array[i]);
-    }
+    newArray = array;
```

Comment:
This isn't correct. If the count is more than the length, you should just be returning an empty array. Since the initial value of `newArray` is an empty array, you can likely remove this block and just make your main block the `else` block by changing your condition to `if(count < array.length){`